f2/src/forms/ConfirmationForm.js

```diff
@@ -42,30 +41,33 @@ export const ConfirmationForm = (props) => {
           <Trans id="analystReport.selfHarmWord" />
         </div>
       ) : null}
-      <Form
-        onSubmit={props.onSubmit}
-        render={({ handleSubmit }) => (
-          <Stack
-            as="form"
-            onSubmit={handleSubmit}
-            shouldWrapChildren
-            spacing={6}
-          >
+      <Formik
+        initialValues={{}}
+        onSubmit={(values) => {
+          props.onSubmit(values)
+        }}
+      >
+        {({ handleSubmit }) => (
+          <Form onSubmit={handleSubmit}>
             {submitted ? (
-              <Well variantColor="blue">
-                <Trans
-                  id="confirmationPage.thankyou"
-                  values={{ reference: reportId }}
-                />
-              </Well>
+                <Card bg="primary">
+                  <Card.Body> 
```

Comment:
For consistency purposes we should use the Info component from f2\src\components\formik\alert\index.js. You can find an example in HowDidItStartForm.js.

             <Row className="form-section">
                <Info>
                  <Trans id="howDidItStartPage.tip" />
                </Info>
              </Row>

Reply:
I think "Card" is a natural replacement of "Well". Bootstrap had component "Well" in the old version, then they dropped it and used "Card" instead(cards replace old panels, wells, and thumbnails). And it looks very similar to the original one.
![confirmation](https://user-images.githubusercontent.com/68352206/91599102-c7a05980-e933-11ea-9cca-eca4071163be.png)

Reply:
The new Info component has been styled to reflect the [Figma design](https://www.figma.com/file/R054FzUOrpP13oTbMsQevj/Sprint-1-%26-2-(Beta-2)?node-id=2082%3A4393). I think it makes sense for all of these messages to have a similar appearance.

Reply:
Sure, I'll change it.

Reply:
OK, changed.